codeship-services.yml

```diff
@@ -7,7 +7,7 @@ db:
         MYSQL_PASSWORD: silauth
 
 web:
-    image: silintl/ssp-base:latest
+    image: silintl/ssp-base:develop
```

Comment:
For tests on Codeship (i.e. this file), wouldn't you want to use `latest` instead of `develop`, since it will be running in production using `latest`?

Reply:
Yes, thanks.